src/player.cpp

```diff
@@ -1781,7 +1781,7 @@ bool player::digging() const
 
 bool player::is_on_ground() const
 {
-    return hp_cur[hp_leg_l] == 0 || hp_cur[hp_leg_r] == 0 || has_effect( effect_downed );
```

Comment:
this one looks more like get_broken_leg_count() instead of get_working_leg_count(). The logic is not right here.

Reply:
Wow thanks you are right I got it backwards.

Reply:
Thanks, I got the logic backwards.